old.Makefile

```diff
@@ -0,0 +1,218 @@
+REPO_ROOT=$(abspath $(dir $(lastword $(MAKEFILE_LIST))))
```

Comment:
Any real reason to keep this file around?  It exists in the git history.

Reply:
Just using it as a reference for build targets, but no, it'll go away before we merge